app/views/admin_general/index.html.erb

```diff
@@ -54,12 +54,17 @@
                                  as requiring administrator attention' } %>
 
     <%= render partial: 'to_do_list',
-               locals: { id: 'unclassified',
-                         parent: 'public-request-things-to-do',
-                         items: @old_unclassified,
-                         label:  "Classify responses that are still unclassified
- #{I18n.with_locale(:en) { distance_of_time_in_words(InfoRequest::OLD_AGE_IN_DAYS)
-    }} after response" } %>
+               locals: {
+                 id: 'unclassified',
+                 parent: 'public-request-things-to-do',
+                 items: @old_unclassified,
+                 label:  "Classify responses that are still unclassified " \
+                         "#{ AlaveteliLocalization.with_locale(:en) do
+                               distance_of_time_in_words(
+                                 InfoRequest.OLD_AGE_IN_DAYS
+                               )
+                            }} after response"
+               } %>
```

Comment:
I think it's ok - weirdly - as it needs to close the `#{` as well as the `do` (and then `locals: {`) but closing `do` with `}` is pretty hard to look at so adopting suggested fix 👍